taxi/views.py

```diff
@@ -1,3 +1,15 @@
 from django.shortcuts import render
+from django.http import HttpRequest, HttpResponse
+from taxi.models import Driver, Car, Manufacturer
 
-# Create your views here.
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
+    context = {
+        "num_drivers": num_drivers,
+        "num_manufacturers": num_manufacturers,
+        "num_cars": num_cars
+    }
```

Comment:
There is no need to create these variables, you could pass expressions as dict values directly